src/app-layout/__integ__/app-layout-toolbar-tooltips.test.ts

```diff
@@ -0,0 +1,337 @@
+              async () => {
+                await page.hoverElement(wrapper.findDrawerTriggerById(nestedDrawerId).toSelector());
+                await expect(page.isExisting(`.${visualRefreshStyles['trigger-tooltip']}`)).resolves.toBeTruthy();
+                await expect(
+                  page.getElementsCount(wrapper.findByClassName(visualRefreshStyles['trigger-tooltip']).toSelector())
+                ).resolves.toBe(1);
+                await page.hoverElement(`.${visualRefreshStyles[`drawers-desktop-triggers-container`]}`);
+                await expect(page.isExisting(`.${visualRefreshStyles['trigger-tooltip']}`)).resolves.toBeFalsy();
+              };
+            }
+
+            await page.click(wrapper.findActiveDrawerCloseButton().toSelector());
+            await expect(page.isExisting(wrapper.findActiveDrawer().toSelector())).resolves.toBeFalsy();
+            await expect(page.isFocused(wrapper.findDrawerTriggerById(drawerId).toSelector())).resolves.toBeTruthy();
+            await expect(page.isExisting(`.${visualRefreshStyles['trigger-tooltip']}`)).resolves.toBeFalsy();
+          };
+        }
+      })
+    );
+
+    test(
+      'tooltip shows on focus and hides on blur events',
+      setupTest({ disableContentPaddings: 'true', visualRefresh: 'true', size }, async page => {
+        await expect(page.isExisting(`.${visualRefreshStyles['trigger-tooltip']}`)).resolves.toBeFalsy();
+        await expect(
+          page.isExisting(`.${visualRefreshStyles[`drawers-desktop-triggers-container`]}`)
+        ).resolves.toBeTruthy();
+        await expect(
+          page.isExisting(`.${visualRefreshStyles['drawers-desktop-triggers-container']}`)
+        ).resolves.toBeTruthy();
+
+        for (const drawerId of drawerIds) {
+          async () => {
+            await page.hoverElement(wrapper.findDrawerTriggerById(drawerId).toSelector());
+            await expect(page.isExisting(`.${visualRefreshStyles['trigger-tooltip']}`)).resolves.toBeTruthy();
+            await expect(
+              page.getElementsCount(wrapper.findByClassName(visualRefreshStyles['trigger-tooltip']).toSelector())
+            ).resolves.toBe(1);
+            await page.hoverElement(`.${visualRefreshStyles[`drawers-desktop-triggers-container`]}`);
+            await expect(page.isExisting(`.${visualRefreshStyles['trigger-tooltip']}`)).resolves.toBeFalsy();
+            await page.click(`button[data-testid='awsui-app-layout-trigger-${drawerId}']`);
+            await expect(page.isExisting(`.${visualRefreshStyles['trigger-tooltip']}`)).resolves.toBeFalsy();
+            await expect(page.isExisting(wrapper.findActiveDrawer().toSelector())).resolves.toBeTruthy();
+            await page.hoverElement(`.${visualRefreshStyles[`drawers-desktop-triggers-container`]}`);
+            await expect(page.isExisting(wrapper.findActiveDrawer().toSelector())).resolves.toBeTruthy();
+
+            for (const nestedDrawerId of drawerIds) {
```

Comment:
Isn't this looping over the same elements as the parent loop? Is this intended?

Reply:
yes. confirming the tooltips work when the drawer is each drawer is opened

Reply:
I was a nice to have but caused tests to time out.  It was robust, signaling that an open drawer did not have an effect on the tooltips but testing against ever trigger was too much. Simplified now